### PR TITLE
Rename ListStoreExtManual to GtkListStoreExtManual

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of https://github.com/gtk-rs/gio/pull/131

cc @EPashkin @sdroege

### CODE REVIEW

File: src/prelude.rs [hunk 1]
```diff
@@ -17,8 +17,8 @@ pub use color_chooser::ColorChooserExtManual;
 pub use dialog::DialogExtManual;
 pub use drag_context::DragContextExtManual;
 pub use fixed::FixedExtManual;
+pub use list_store::GtkListStoreExtManual;
 pub use invisible::InvisibleExtManual;
-pub use list_store::ListStoreExtManual;
 pub use menu::MenuExtManual;
```
Comment (EPashkin) [inline src/prelude.rs]: IMHO move line broke sort

**Comment by EPashkin** [comment]
@GuillaumeGomez Please remove `ListStoreExt` too (like https://github.com/gtk-rs/gtk/blob/master/Gir.toml#L417)

**Comment by EPashkin** [comment]
*rename (not remove)

File: src/prelude.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply src/prelude.rs]: ...FGHI... -> Seems good to me. :)

**Comment by GuillaumeGomez** [comment]
Oh nice! I didn't know we had this option. Makes things much more easy!

**Comment by EPashkin** [comment]
Thanks.

**Pull request merged**